main.js

```diff
@@ -21,4 +21,146 @@ closebutton.addEventListener('click', closeMenu);
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+  {
+    id: 4,
+    name: 'Multi-Post Stories',
+    image: 'img/work_img/Portfolio3.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+];
+
+const projectObj = {
+  work1: {
+    id: 1,
+    name: 'Multi-Post Stories',
+    image: 'img/work_img/Portfolio.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+  work2: {
+    id: 2,
+    name: 'Multi-Post Stories',
+    image: 'img/work_img/Portfolio1.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+  work3: {
+    id: 3,
+    name: 'Multi-Post Stories',
+    image: 'img/work_img/Portfolio2.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+  work4: {
+    id: 4,
+    name: 'Multi-Post Stories',
+    image: 'img/work_img/Portfolio3.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required.has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tech: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    live: 'https://chuanqizhu0820.github.io/My-Portfolio/',
+    source: 'https://github.com/chuanqizhu0820/My-Portfolio',
+  },
+};
+
+const workcontainer = document.querySelector('.work-container');
+
+let projhtml = '';
+
+projectInfo.forEach((item) => {
```

Comment:
- [ ] The given design specs from figma have not been met. Please ensure that the popup window match the given specs.
 
![Screenshot from 2021-09-03 14-06-36](https://user-images.githubusercontent.com/65501535/131996017-a4a38383-9c2e-4958-89ff-751c1aba5303.png)

Reply:
Hi, the popup window does not look like this actually. Could you check it once again? I have already made changes according to the first reviewer's suggestions.